inheritance/lib/src/main/java/inheritance/Restaurant.java

```diff
@@ -0,0 +1,111 @@
+package inheritance;
+
+import javax.annotation.Nullable;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.NoSuchElementException;
+
+public class Restaurant {
```

Comment:
I like the setup for your restaurant class, looks good so far! a hashmap for reviews!